### PR TITLE
Apple Clone - Gradients and image Backgrounds

### PR DESCRIPTION
**Pull request by dansantander**
I cloned the apple page using gradients and image backgrounds. Also, I used flex, grid and float for fitting the elements.

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @dansantander 

### Good job this far you have come. However, there are some changes you need to work on before your project can be approved.
- Make your banner text visible, they are hidden behind the image thumbnails
text
- Make your footer design better, refer the original [website](https://web.archive.org/web/20140301004610/http://www.apple.com/)

### Once you have made these required changes, resubmit for another review. I wish you all the best in the next submission.
- Happy coding!
- For Help: Contact me @rOluochKe

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: style.css
Comment (addod19) [inline style.css:11]: Kindly fix the horizontal scroll by setting the ``` .container { width: 100% }``` to fit on all screen size

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @dansantander, 💯 congratulations on a great job done. However, there is a horizontal scroll to fix. 😄 
Kindly make this change and submit a code review. 👍 

Happy coding 💻 
Daniel Larbi Addo

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**
